build/lib/mqtt2kasa/bin/create-env.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Hmm... I am hoping we could move the files here, instead of duplicating them. If that is intentional, then maybe we could consider using symbolic links. Let me thinks a bit more about this.